libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +267,167 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+#if AP_TERRAIN_AVAILABLE
+// calculate poi information
+void AP_Camera_Backend::calculate_poi()
```

Comment:
I think we should move the calculate_poi to AP_Mount.  It probably makes sense to move the thread as well to AP_Mount.  The send_camera_fov_status() should stay here in AP_Camera though.